Documentation/user-documentation/user-documentation.md

```diff
@@ -0,0 +1,115 @@
+# SOSCI User Documentation
+
+## Table of contents
+
+- [Getting started](#getting-started)
+    - [How to access the frontend and backend?](#how-to-access-the-frontend-and-backend)
+    - [How to send samples via UDP?](#how-to-send-samples-via-udp)
+- [Using the frontend](#using-the-frontend)
+    - [General](#general)
+    - [Top control panel](#top-control-panel)
+    - [Oscilloscope](#oscilloscope)
+    - [Bottom control panel](#bottom-control-panel)
+    - [Settings](#settings)
+
+## Getting started
```

Comment:
How about adding a brief introduction to explain what sosci is and its potential uses? That could be helpful.

Reply:
Don't we have the product vision and mission for that?
Maybe also add it in here?

Reply:
I might be mistaken, but here's my understanding of what user documentation should be like. As a user of the software, it should be the only resource I need to consult. It should contain an introduction to the software and all the features and functionality it provides.

Reply:
done